packages/block-editor/src/components/block-preview/index.js

```diff
@@ -23,22 +23,23 @@ import BlockList from '../block-list';
  *
  * @return {WPElement} Rendered element.
  */
-function BlockPreview( props ) {
-	return (
-		<div className="editor-block-preview block-editor-block-preview">
-			<div className="editor-block-preview__title block-editor-block-preview__title">{ __( 'Preview' ) }</div>
-			<BlockPreviewContent { ...props } />
-		</div>
-	);
-}
-
-export function BlockPreviewContent( { blocks, settings } ) {
+function BlockPreview( { blocks, settings, className, isScaled } ) {
 	if ( ! blocks ) {
 		return null;
 	}
-
 	return (
-		<Disabled className="editor-block-preview__content block-editor-block-preview__content editor-styles-wrapper" aria-hidden>
+		<Disabled
+			aria-hidden
+			className={
+				classnames(
+					'editor-block-preview', 'block-editor-block-preview',
```

Comment:
I think since we changed the "purpose of the component" anyway, we may want to drop "editor-block-preview"?

Reply:
Good, I thought it might be the case 👍